app/views/works/_standard_form.html.erb

```diff
@@ -288,7 +288,7 @@
           <%=
             radio_button_list(f, :comment_permissions, [
               [:enable_all, t("comments.commentable.permissions.options.enable_all")],
-              [:disable_anon, t("comments.commentable.permissions.options.disable_anon")],
+              [:disable_anon, t("comments.commentable.permissions.options.disable_anon"), checked: @work.new_record? || @work.comment_permissions == "disable_anon" ],
```

Comment:
I believe this will override the user's selection if they try to create a new work allowing guest comments, but get an error when trying to post (e.g. if they don't pick a warning). I'd suggest either using the `_default` option when defining the `enum` in the work class, or changing the default on the table column using a migration, rather than trying to override that default value here to enforce a different default.

(Changes to the default are metadata-only, so if you do take the migration approach I don't think you have to worry about `pt-online-schema-change`.)

Reply:
I think I'll try the `_default` way, thanks!